components/vocabularies/chebi/api/src/main/resources/chebi/conf/schema.xml

```diff
@@ -1,4 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+-->
+
 <!--
```

Comment:
There's a double licence in this file. We only do AGPLv3 now, just run `mvn license:remove` followed by `mvn license:format` to update the licenses.